aten/src/ATen/core/aten_interned_strings.h

```diff
@@ -599,6 +599,7 @@ _(aten, rnn_tanh) \
 _(aten, rnn_tanh_cell) \
 _(aten, rot90) \
 _(aten, round) \
+_(aten, row_stack) \
```

Comment:
Since row_stack is an alias, see the instructions for adding an alias here:

https://github.com/pytorch/pytorch/blob/ec5f81f9d334efa3d46ca9c66885e2ab14b6ce01/aten/src/ATen/native/native_functions.yaml#L236

Its entry needs to go in aten/src/ATen/core/interned_strings.h and torch/csrc/jit/passes/normalize_ops.cpp needs an update, too.

Reply:
Thanks for the notes on adding aliases. the `vstack` entry is removed from `aten/src/ATen/core/aten_interned_strings.h` and both `vstack` and `row_stack` entries are added in the `aten/src/ATen/core/interned_strings.h`.
`torch/csrc/jit/passes/normalize_ops.cpp` is updated as well.